<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="PresetPositionGroupA_PB">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PresetPositionGroupB_PB">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>190</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PresetPositionGroupD_PB">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PresetPositionGroupC_PB">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PresetPosition2_PB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/presetposition2.png</normaloff>:/icons/presetposition2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="PresetPosition1_PB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/presetposition1.png</normaloff>:/icons/presetposition1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="RinsingPosition_PB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/rinsingposition.png</normaloff>:/icons/rinsingposition.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="PresetPosition3_PB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/presetposition3.png</normaloff>:/icons/presetposition3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BowlRinsing_PB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/bowlrinsing.png</normaloff>:/icons/bowlrinsing.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="CupFiller_PB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/cupfiller.png</normaloff>:/icons/cupfiller.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="OperatingLight_PB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/operatinglight.png</normaloff>:/icons/operatinglight.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="WaterHeater_PB">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/waterheater.png</normaloff>:/icons/waterheater.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="Unit_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>350</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/icons/dentalunit.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset_PB">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Setting_PB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Serif</family>
      <pointsize>41</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChairUpward_PB">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/chairupward.png</normaloff>:/icons/chairupward.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ChairDownward_PB">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/chairdownward.png</normaloff>:/icons/chairdownward.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BackrestForward_PB">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/backrestforward.png</normaloff>:/icons/backrestforward.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BackrestBackward_PB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/backrestbackward.png</normaloff>:/icons/backrestbackward.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="CallAssist_PB">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/callassist.png</normaloff>:/icons/callassist.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>70</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="FilmViewer_PB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/filmviewer.png</normaloff>:/icons/filmviewer.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <zorder>Unit_Label</zorder>
   <zorder>PresetPositionGroupA_PB</zorder>
   <zorder>PresetPositionGroupB_PB</zorder>
   <zorder>PresetPositionGroupD_PB</zorder>
   <zorder>PresetPositionGroupC_PB</zorder>
   <zorder>PresetPosition2_PB</zorder>
   <zorder>PresetPosition1_PB</zorder>
   <zorder>RinsingPosition_PB</zorder>
   <zorder>PresetPosition3_PB</zorder>
   <zorder>BowlRinsing_PB</zorder>
   <zorder>CupFiller_PB</zorder>
   <zorder>OperatingLight_PB</zorder>
   <zorder>WaterHeater_PB</zorder>
   <zorder>Reset_PB</zorder>
   <zorder>Setting_PB</zorder>
   <zorder>ChairUpward_PB</zorder>
   <zorder>ChairDownward_PB</zorder>
   <zorder>BackrestForward_PB</zorder>
   <zorder>BackrestBackward_PB</zorder>
   <zorder>CallAssist_PB</zorder>
   <zorder>FilmViewer_PB</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
